tortoise/__init__.py

```diff
@@ -345,4 +348,32 @@ def _get_config_from_config_file(cls, config_file):
         await cls._reset_connections()
 
 
+def run_async(coro):
+    """
+    Simple async runner that cleans up DB connections on exit.
+    This is meant for simple scripts.
+
+    Usage::
+
+        from tortoise import Tortoise, run_async
+
+        async def do_stuff():
+            await Tortoise.init(
+                db_url='sqlite://db.sqlite3',
+                models={'models': ['app.models']}
+            )
+
+            ...
+
+        run_async(do_stuff())
+    """
+    loop = asyncio.get_event_loop()
+    try:
+        loop.run_until_complete(coro)
+    except Exception as exc:
+        raise
```

Comment:
Why is it needed to catch exception just to reraise it?

Reply:
can you do a try → finally without an except in Python?

Reply:
Answer is "yes". ok, I'll fix that,